src/components/input/search-input.jsx

```diff
@@ -0,0 +1,37 @@
+import ctl from "@netlify/classnames-template-literals";
+import React from "react";
+
+import { Input } from "./input";
+import SearchIcon from "assets/images/svgs/search.svg";
+
+const SearchInput = ({ placeholder, value, onChange = () => {} }) => {
```

Comment:
Why I like the idea of this component being reusable. I think it's an 'early' abstraction as we cannot predict if there will be more search inputs in the future and by the time it's needed, we might be struggling to make the component do a lot. It 'works' but let's move it closer to where it's used right now. cc @Joponpin2000 @Demoivre02